docroot/sites/all/conf/environment.inc

```diff
@@ -1,31 +1,39 @@
 <?php
 
+const D_ENVIRONMENT_LOCAL = 'local';
+const D_ENVIRONMENT_DEV = 'dev';
+const D_ENVIRONMENT_TEST = 'test';
+const D_ENVIRONMENT_PROD = 'prod';
+
+const D_PLATFORM_DOCKER = 'docker';
+const D_PLATFORM_PIPELINES = 'pipelines';
+const D_PLATFORM_ACQUIA = 'acquia';
+const D_PLATFORM_PANTHEON = 'pantheon';
+const D_PLATFORM_PLATFORMSH = 'platformsh';
+
 $base_domains = array(
 
   // Docker
-  getenv('D4D_HOSTNAME') => 'local',
+  getenv('D4D_HOSTNAME') => D_ENVIRONMENT_LOCAL,
 
 );
 
 $platform = '';
 $instance = $_SERVER['HTTP_HOST'];
 $env = $base_domains[$_SERVER['HTTP_HOST']];
+$acquia_settings_file = '/var/www/site-php/my_project/my_project-settings.inc';
```

Comment:
wouldn't this be better within the acquia block below?

Reply:
Yeah, I agree with that

Reply:
I've updated the file